interface/VideoEncoderDefs.h

```diff
@@ -153,11 +153,8 @@ typedef struct VideoFrameRate {
     uint32_t frameRateDenom;
 }VideoFrameRate;
 typedef struct VideoTemproalLayers {
-    //how many layers higher than layer 0
-    //layer 0 framerate is VideoParamsCommon.frameRate
-    //layer 0 bitrate is VideoParamsCommon.rcParams.bitRate
-    //layer 1 framterate is frameRate[0]
-    //layer 1 birate is bitRate[0]
+    //framerate of the highest layer is VideoParamsCommon.frameRate
+    //bitrate of the highest layer is VideoParamsCommon.rcParams.bitRate
     uint8_t numLayers;
     VideoFrameRate frameRate[32];
```

Comment:
since we do not need user set this any more, could we move this to VaapiEncBase?